code/solr/tasks/Solr_Reindex.php

```diff
@@ -65,7 +70,7 @@ protected function doReindex($request)
         // this is for when index names do not match the class name (this can be done by overloading getIndexName() on
         // indexes
         if ($index && !ClassInfo::exists($index)) {
-            foreach(ClassInfo::subclassesFor('SolrIndex') as $solrIndexClass) {
+            foreach(ClassInfo::subclassesFor('SilverStripe\FullTextSearch\Solr\SolrIndex') as $solrIndexClass) {
```

Comment:
Ah, I see yeah, it is used in tandem with `use` for namespaced classes.
Anyway, I have updated my pull request for you to review a little bit more.